recipes-bsp/u-boot/u-boot-envs-atmel.inc

```diff
@@ -1,16 +1,20 @@
 # U-Boot Environments for AT91 boards
 
-SRC_URI_append = " file://*.txt"
+SRC_URI_append = " file://envs/"
 
 ENV_SIZE = "0x4000"
 ENV_FILENAME = "uboot.env"
 ENV_FILEPATH = "${WORKDIR}/envs"
 
-do_deploy_append() {
+do_compile_append() {
     if [ -e "${ENV_FILEPATH}/${MACHINE}.txt" ]; then
-        echo "Compile u-boot environment for ${MACHINE} and deploy it."
+        echo "Compile U-Boot environment for ${MACHINE} and deploy it."
         ${B}/tools/mkenvimage -s ${ENV_SIZE} ${ENV_FILEPATH}/${MACHINE}.txt -o ${ENV_FILENAME}
     else
         echo "No custom environment available for ${MACHINE}."
     fi
 }
+
+do_deploy_append() {
+	install -Dm 0644 ${B}/${ENV_FILENAME} ${DEPLOYDIR}
```

Comment:
This will generate an error if no uboot.env file is generated.  Would something like this work?
```suggestion
     if [ -e  ${B}/${ENV_FILENAME}  ]
	    install -Dm 0644 ${B}/${ENV_FILENAME} ${DEPLOYDIR}
```

Reply:
Indeed, fixed now.